app/controllers/idv/account_verified_cta_visited_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Idv
+  class AccountVerifiedCtaVisitedController < ApplicationController
+    before_action :disable_caching
+    before_action :validate_redirect_url_exists
+
+    rescue_from ActionController::ActionControllerError, with: :render_not_acceptable
```

Comment:
A/N: I hate this. Not sure what we want to actually do here.

Reply:
Would it make any sense to add an error page saying, "Oops! We tried to redirect you to `#{sp_name}` but we don't have a link to send you to!" or something? I feel like that way at least the user would understand that a technical error had occurred and it wasn't their fault?

Reply:
Yeah, I think it makes sense to circle back with @artfulaction on this question. I don't want to just give a generic error page without contextual information.